### ISSUE TITLE
UnicodeEncodeError is raised for VireoSheet.createFromExcelFile

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/dspace-python#21]
The following errors are raised when attempting to parse Excel restrictions spreadsheets:

```
*** UnicodeEncodeError: 'ascii' codec can't encode character u'\u2019' in position 5: ordinal not in range(128)
```

This originates from https://github.com/pulibrary/dspace-python/blob/thesis-central-import-multiauthor/thesiscentral-vireo/dataspace/python/vireo.py#L77

**Comment by jrgriffiniii** [comment]
This was resolved with f67b083, but I have not yet issued a pull request for this branch.

**Comment by kmcelwee** [comment]
Shifting to icebox for now until VireoSheet is deprecated.
